pkg/executor/infoschema_reader_test.go

```diff
@@ -385,7 +394,7 @@ func TestForServersInfo(t *testing.T) {
 	require.Equal(t, info.Lease, rows[0][4])
 	require.Equal(t, info.Version, rows[0][5])
 	require.Equal(t, info.GitHash, rows[0][6])
-	require.Equal(t, stringutil.BuildStringFromLabels(info.Labels), rows[0][8])
+	require.Equal(t, "dc=dc1", rows[0][7])
```

Comment:
@lance6716  Can we get the column by name instead of by number? That might be a better test for this issue.

Reply:
seems that the TestKit utils we used in unit tests does not return column names.

Reply:
While the `Result` doesn't have the fields, the `sqlexec.RecordSet` does. Maybe we can do it like this:
https://github.com/pingcap/tidb/pull/59259